src/inference/include/openvino/runtime/auto/properties.hpp

```diff
@@ -28,5 +28,50 @@ static constexpr Property<bool> enable_startup_fallback{"ENABLE_STARTUP_FALLBACK
  * selected device
  */
 static constexpr Property<bool> enable_runtime_fallback{"ENABLE_RUNTIME_FALLBACK"};
+
+/**
+ * @brief Enum to define schedult policy of inference request for MULTI
```

Comment:
```suggestion
 * @brief Enum to define schedule policy of inference request for MULTI
```
MULTI is deprecated and we need to refer as AUTO in tput mode

Reply:
Updated.